contracts/protocol/libraries/position/LiquidityBitmap.sol

```diff
@@ -78,15 +78,28 @@ library LiquidityBitmap {
     ) internal view returns (uint128 next) {
         uint128 startWord = pip >> 8;
         if (lte) {
-            for (
-                uint128 i = startWord;
-                i > (startWord < maxWords ? 0 : startWord - maxWords);
-                i--
-            ) {
-                if (self[i] != 0) {
+            if (startWord != 0) {
```

Comment:
add condition if startWord != 0. if startWord == 0 we just need to find liquidity in word 0, no need for loop multiple words